linodecli/api_request.py

```diff
@@ -283,6 +283,9 @@ def _print_request_debug_info(method, url, headers, body):
     """
     print(f"> {method.__name__.upper()} {url}", file=sys.stderr)
     for k, v in headers.items():
+        # If this is the Authorization header, sanitize the token
+        if k.lower() == "authorization":
+            v = "Bearer *******************************"
```

Comment:
nitpick: We can use `"*" * 64` to simplify this string literal a tiny bit

Reply:
Fixed!